lib/coroutine.c

```diff
@@ -248,6 +248,18 @@ M0_INTERNAL int m0_co_op_tick_ret(struct m0_co_op *op,
 }
 M0_EXPORTED(m0_co_op_tick_ret);
 
+M0_INTERNAL void m0_co_op_wait(struct m0_co_op *op)
+{
+	struct m0_sm *sm = &op->co_sm;
+	int           rc;
+
+	m0_sm_group_lock(&op->co_sm_group);
+	rc = m0_sm_timedwait(sm, M0_BITS(COR_DONE), M0_TIME_NEVER);
```

Comment:
Please note this is a wait with M0_TIME_NEVER time value.
This might cause HUNG issue.
So, we need to make sure this function is used properly.

Reply:
We had a discussion with Max about this patch, and I decided to remove this commit. It will go as a separate PR.

Reply:
OK